suite-native/module-check-backup/src/components/CheckBackupTutorialStep2.tsx

```diff
@@ -30,28 +27,18 @@ type NavigationProps = StackNavigationProps<
     DeviceCheckBackupStackRoutes.CheckBackupTutorial
 >;
 
-const checkBackupUnsupportedDeviceModels: Array<DeviceModelInternal> = [DeviceModelInternal.T1B1];
-
 export const CheckBackupTutorialStep2 = ({
     currentStepIndex,
 }: WalletBackupTutorialNumberedStepProps) => {
     const navigation = useNavigation<NavigationProps>();
-    const deviceModel = useSelector(selectDeviceModel);
-
-    const { applyStyle } = useNativeStyles();
-    const { showToast } = useToast();
 
     const navigateToCheckBackup = () => {
-        if (deviceModel && checkBackupUnsupportedDeviceModels.includes(deviceModel)) {
-            navigation.navigate(DeviceCheckBackupStackRoutes.UnsupportedModel, {
-                deviceModel: models[deviceModel].name,
-            });
-
-            return;
-        }
         navigation.navigate(DeviceCheckBackupStackRoutes.CheckBackup);
     };
 
+    const { applyStyle } = useNativeStyles();
+    const { showToast } = useToast();
```

Comment:
Prefet to keep the hooks initialization above any methods etc please. I know it's not using any built-in hooks from react-native, so linter doesn't mind, but I think this is more stylistic thing for component organization. Not a big issue, feel free to resolve